packages/plugins/RedPacket/src/SiteAdaptor/RedPacketDialog.tsx

```diff
@@ -251,7 +251,16 @@ export default function RedPacketDialog(props: RedPacketDialogProps) {
             !showHistory &&
             !showDetails
         ) {
-            return <Icons.History onClick={() => setShowHistory((history) => !history)} />
+            return (
+                <Icons.History
+                    style={{ cursor: account ? undefined : 'disabled' }}
```

Comment:
`disabled` not a valid value.

https://developer.mozilla.org/en-US/docs/Web/CSS/cursor